Flaky CI: File exist error

```
_ TestOnnxBackEnd.test_export2python_produces_correct_onnx_script_model_0050_test_log_example _
[gw1] darwin -- Python 3.8.16 /Users/runner/work/onnx-script/onnx-script/.nox/test/bin/python
.nox/test/lib/python3.8/site-packages/parameterized/parameterized.py:533: in standalone_func
    return func(*(a + p.args), **p.kwargs)
onnxscript/backend/onnx_export_test.py:178: in test_export2python_produces_correct_onnx_script_model
    functions = extract_functions(backend_test.name, code, self.test_folder)
onnxscript/backend/onnx_export_test.py:99: in extract_functions
    test_folder.mkdir()
../../../hostedtoolcache/Python/3.8.16/x64/lib/python3.8/pathlib.py:1288: in mkdir
    self._accessor.mkdir(self, mode)
E   FileExistsError: [Errno 17] File exists: '/Users/runner/work/onnx-script/onnx-script/onnxscript/tests/onnx_backend_test_code'
```

Need to use exsit_ok etc.

--- a/onnxscript/backend/onnx_export_test.py
+++ b/onnxscript/backend/onnx_export_test.py
@@ -96,9 +96,9 @@ def run_function(obj, *inputs):
 def extract_functions(name: str, content: str, test_folder: pathlib.Path):
     """Write the content into a file and import all OnnxFunctions from it."""
     if not test_folder.exists():
-        test_folder.mkdir()
+        test_folder.mkdir(exist_ok=True, parents=True)
         init = test_folder / "__init__.py"
-        init.touch()
+        init.touch(exist_ok=True)
     file = test_folder / f"{name}.py"
     file.write_text(content, encoding="utf-8")
 

